tensorrt_llm/serve/scripts/benchmark_serving.py

```
@@ -659,6 +709,29 @@ def save_to_pytorch_benchmark_format(args: argparse.Namespace,
+    """
+    energy_url = f"{base_url}/energy_metrics"
+
+    async with aiohttp.ClientSession(trust_env=True,
+                                     timeout=AIOHTTP_TIMEOUT) as session:
+        try:
+            async with session.get(energy_url) as response:
+                if response.status == 200:
+                    return await response.json()
+                else:
+                    return None
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)